contracts/upgradeable_contracts/BaseBridgeValidators.sol

```diff
@@ -51,7 +51,7 @@ contract BaseBridgeValidators is EternalStorage, Ownable {
             index = next;
             next = getNextValidator(index);
 
-            if (next == F_ADDR) {
+            if (next == F_ADDR || next == address(0) ) {
```

Comment:
I would say that we need to check the return value after the lines 37, 48 and 91.

Reply:
Done, added those checks here https://github.com/poanetwork/poa-bridge-contracts/pull/130/commits/5bf54405c98eba14c525e42d7627595f170189fe